### PR TITLE
chore(gems): upgrade fluentd to allow v1.13.0

### PR DESCRIPTION
**Pull request by Cryptophobia**
- add up to v1.14 fluentd gem
 - fluentd [v1.13.0 changelog](https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v1130---20210529)

Signed-off-by: Anton Ouzounov <aouzounov@vmware.com>

### CODE REVIEW

**Comment by Cryptophobia** [comment]
@jcantrill , this PR is to allow upgrade to latest fluentd release v1.13.0. Can we merge this is in to allow fluentd to be upgrade. Changelog here: https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v1130---20210529

File: fluent-plugin-kubernetes_metadata_filter.gemspec [hunk 1]
```diff
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |gem|
   gem.name          = 'fluent-plugin-kubernetes_metadata_filter'
-  gem.version       = '2.7.0'
+  gem.version       = '2.7.1'
```
Comment (jcantrill) [inline fluent-plugin-kubernetes_metadata_filter.gemspec:8]: Please revert this change otherwise lgtm

**Review by jcantrill — CHANGES_REQUESTED** [review]

File: fluent-plugin-kubernetes_metadata_filter.gemspec (hunk 1 above)
Comment (Cryptophobia) [reply fluent-plugin-kubernetes_metadata_filter.gemspec]: Oops, good point. This is backwards compatible so no need to push gem version forward.

**Comment by Cryptophobia** [comment]
@jcantrill , keeping gem version at 2.7.0, thanks for the quick review!

**Comment by Cryptophobia** [comment]
@jcantrill , can you take a look again when you have some time? thank you.

**Comment by jcantrill** [comment]
published to 2.7.1
